packages/venia-ui/lib/components/Footer/footer.js

```diff
@@ -25,11 +25,11 @@ const Footer = props => {
             const itemKey = `text: ${text} path:${path}`;
             const child = path ? (
                 <Link className={classes.link} to={path}>
-                    <FormattedMessage id={text} />
+                    <FormattedMessage id={text} defaultMessage={text} />
```

Comment:
Does this text come directly from the backend? This will be hard to adjust/modify without some way to correlate to the translation file.

Reply:
Interestingly, it is pointing to a simple `sampleData.js` file